packages/airnode-node/src/coordinator/calls/coordinated-execution.test.ts

```diff
@@ -28,7 +30,13 @@ describe('callApis', () => {
 
   it('returns each API call with the response if successful', async () => {
     const config = fixtures.buildConfig();
-    jest.spyOn(fs, 'readFileSync').mockReturnValueOnce(JSON.stringify(config));
+    jest.spyOn(fs, 'readFileSync').mockImplementation((...args) => {
```

Comment:
Can you define this function in https://github.com/api3dao/airnode/blob/master/packages/airnode-node/test/mock-utils.ts and import it in the tests?

Reply:
You can create similar file in deployer `test` folder.